frontend/pageDesignspecs/mentorPg.css

```diff
@@ -0,0 +1,55 @@
+.jumbotron1{
+    Background-color:#ffffff;
+    margin-top:15px;
+    padding: 20px;
+}
+h1{
+    Background-color:#ffffff;
+    color: black;
+    Font-family: Roboto, sans-serif;
+    Font-size: 32px;
+    margin: 30px;
+    padding-top:10px;
+}
+.stuCard{
+
+}
+.progressBarr{
+    Background-color:#ffffff;
+    color: black;
```

Comment:
Try and pick a color standard - it's weird to have two different ways of writing the same color right next to each other